R/convert_mae_se_assay_to_dt.R

```diff
@@ -113,7 +113,7 @@ convert_se_assay_to_dt <- function(se,
   checkmate::assert_string(assay_name)
   
   object <- assays(se)[[assay_name]]
-  checkmate::assert_true(inherits(object, "BumpyDataFrameMatrix") || inherits(object, "matrix"))
+  checkmate::assertClass(object, "BumpyDataFrameMatrix")
```

Comment:
Are you sure we don't want to have `matrix` as se?
take a look: 
- https://github.com/gdrplatform/gDRutils/blob/GDR-2199/tests/testthat/test-convert_mae_se_assay_to_dt.R#L9
- https://github.com/gdrplatform/gDRutils/blob/GDR-2199/tests/testthat/test-convert_mae_se_assay_to_dt.R#L95

Reply:
fixed